web/blog/vast-v2.5/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: VAST v2.5
+authors: dominiklohmann
+date: 2023-01-20
+tags: [release, cef]
```

Comment:
Next release is gonna be 3.0; also still needs `title: FIXME` and `draft: true`.

Not sure if I'm gonna be the author, but I suppose we can change that in January when the release will likely come out.

Reply:
Good catch.